LSP-kotlin.sublime-settings

```diff
@@ -0,0 +1,6 @@
+{
+	"command": [
+		"${storage_path}/LSP-kotlin/server/bin/kotlin-language-server"
+	],
+	"selector": "source.Kotlin"
+}
```

Comment:
It's safer to keep the empty `settings` object. Otherwise `didChangeConfiguration` notification will not be sent and that can change server behavior if it only processes settings on getting the notification.